common/Fabric-Install-Utilities.psm1

```diff
@@ -84,11 +84,27 @@ function New-AppRoot($appDirectory, $iisUser){
 }
 
 function New-AppPool($appName, $userName, $credential){
-    Set-Location IIS:\AppPools
+    Set-Location "IIS:\AppPools"
     if(!(Test-Path $appName -PathType Container))
     {
         Write-Console "AppPool $appName does not exist...creating."
-        $appPool = New-WebAppPool $appName
+        $successful = $FALSE
```

Comment:
should we use $false here? I believe PowerShell is case sensitive...

Reply:
I saw this syntax in a web page I was looking at while working on this. It seemed to work, but I can change it so it looks better.